src/models/Artifact.ts

```diff
@@ -58,6 +62,7 @@ export class Artifact implements IArtifact {
   public type: ArtifactType
   public description: string | null
   public data: ArtifactData
+  public readonly serializedData: string | null | undefined
```

Comment:
serializedData is basically whatever "raw" data was returned from the API. Different artifacts have different types returned for the `data` field that we need parse to do anything meaningful with. Rather than require the rest of the application to care about that nuance, I'm storing a reference to the un-parsed data in case it's needed (currently only the raw data view uses this)